howso/update_cases.amlg

```diff
@@ -483,6 +489,11 @@
 					))
 					closest_cases_map
 				)
+
+				;also add the weight accumulated to each case to !dataMassChangeSinceLastAnalyze to ensure that cases trained as
+				; only weights (whether through auto-ablation or otherwise) contribute to the progress towards the next auto-analyze,
+				; if enabled.
+				(accum_to_entities (assoc !dataMassChangeSinceLastAnalyze 1))
```

Comment:
should this accum to a model parameter inside a map statement that iterates over all the cases be moved outside of the map statement?